animation_nodes/nodes/color/c_utils.pyx

```diff
@@ -0,0 +1,69 @@
+import cython
+from ... math cimport (
+    addColor,
+    scaleColor_Inplace
+)
+from ... data_structures cimport (
+    Color,
+    LongList,
+    ColorList,
+    VirtualColorList,
+    PolygonIndicesList
+)
+
+def getLoopColorsFromVertexColors(PolygonIndicesList polygons, VirtualColorList colors):
+    cdef loopsCount = len(polygons.indices)
+    cdef ColorList loopsColors = ColorList(length = loopsCount)
+
+    cdef long i
+    for i in range(loopsCount):
+        loopsColors.data[i] = colors.get(polygons.indices[i])[0]
+    return loopsColors
+    
+def getLoopColorsFromPolygonColors(PolygonIndicesList polygons, VirtualColorList colors):
+    cdef long i, j, index
```

Comment:
Move this line just before the for loop. To make declaration and use of the variable closer together.

Reply:
Moved the declaration of variables.